tests/integration/components/observation-form/component-test.js

```diff
@@ -1,26 +1,94 @@
 import { module, test } from 'qunit';
 import { setupRenderingTest } from 'ember-qunit';
-import { render } from '@ember/test-helpers';
+import {
+  render,
+  click,
+  fillIn,
+  findAll,
+  find,
+  waitFor,
+} from '@ember/test-helpers';
 import { hbs } from 'ember-cli-htmlbars';
+import { setupMirage } from 'ember-cli-mirage/test-support';
+import { Interactor as Pikaday } from 'ember-pikaday/test-support';
+import { close as closePikaday } from 'ember-pikaday/test-support';
+import moment from 'moment';
+import { setupMapTest } from 'ember-google-maps/test-support';
 
 module('Integration | Component | observation-form', function (hooks) {
   setupRenderingTest(hooks);
+  setupMirage(hooks);
+  setupMapTest(hooks);
 
-  test.skip('it renders', async function (assert) {
-    // Set any properties with this.set('myProperty', 'value');
-    // Handle any actions with this.set('myAction', function(val) { ... });
+  hooks.beforeEach(async function () {
+    const observation = this.server.create('observation');
+    this.set('observation', observation);
 
     await render(hbs`<ObservationForm />`);
+  });
+
+  test('inputs renders', async function (assert) {
+    assert.dom('[data-test-input-pikaday]').exists();
+    assert.dom('[data-test-input-birdname]').exists();
+    assert.dom('[data-test-input-location]').exists();
+    assert.dom('[data-test-input-notes]').exists();
+    assert.dom('[data-test-button-cancel]').exists();
+    assert.dom('[data-test-button-clear]').exists();
+    assert.dom('[data-test-button-save]').exists();
+  });
+
+  test('pikaday input', async function (assert) {
+    await click('[data-test-input-pikaday]');
+    await Pikaday.selectDate(new Date(2022, 8, 28));
+    assert.strictEqual(Pikaday.selectedYear(), '2022');
+    assert.strictEqual(Pikaday.selectedMonth(), '8');
+    assert.strictEqual(Pikaday.selectedDay(), '28');
+    await closePikaday('[data-test-input-pikaday]');
+  });
+
+  test('location input disabled', async function (assert) {
+    assert.dom('[data-test-input-location]').hasNoAttribute('disabled');
+  });
 
-    assert.dom(this.element).hasText('');
+  test('location input change to disabled', async function (assert) {
+    await click('[data-test-input-location]');
+    assert.dom('[data-test-input-location]').hasAttribute('disabled');
+  });
+
+  test('fill in the form', async function (assert) {
+    assert.dom('[data-test-button-save]').hasAttribute('disabled');
+
+    const stringDate = moment(this.observation.observationDate).format(
+      'YYYY-MM-DD'
+    );
```

Comment:
how about destructure `observation` for each cases:
```suggestion
   const { observationDate, birdname, notes } = this.observation;
    const stringDate = moment(observationDate).format(
      'YYYY-MM-DD'
    );
```